### ISSUE TITLE
Unexpected output for rightshift assignment

### ISSUE THREAD

**Issue by elasmojs** [boa-dev/boa#618]
Hi

```js
let a = 5;      //  00000000000000000000000000000101
a >>= 2;        //  00000000000000000000000000000001
console.log(a); // expected output: 1, got: 20

let b = -5;     // -00000000000000000000000000000101
b >>= 2;        // -00000000000000000000000000000010
console.log(b); // expected output: -2, got: -20
```

thanks

**Comment by HalidOdat** [comment]
Hi @elasmojs , thanks for reporting theese issues.

This has been fixed on master branch when #520 landed ;)

Would it be possible to use [Code blocks](https://docs.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks) in comment so its more readable , and state what version of boa you are tesing this on example `v0.9`, `master` (latest)

**Comment by HalidOdat** [comment]
This has been fixed so closing this issue.
